internal/logging/logger.go

```diff
@@ -2,205 +2,104 @@
 package logging
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"log"
+	"io"
 	"os"
-	"strings"
+	"strconv"
+	"time"
 
 	"github.com/gin-gonic/gin"
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
-	"golang.org/x/term"
-	"gopkg.in/natefinch/lumberjack.v2"
-
-	"github.com/infrahq/infra/internal/server/models"
+	"github.com/rs/zerolog"
 )
 
 var (
-	level                    = zap.NewAtomicLevel()
-	L                        = newLogger(level, os.Stderr)
-	S     *zap.SugaredLogger = L.Sugar()
+	L = NewLogger(zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.RFC3339})
 )
 
-// SetLevel of the global loggers L and S.
-func SetLevel(l string) error {
-	return level.UnmarshalText([]byte(l))
+type logger struct {
+	zerolog.Logger
 }
 
-func newLogger(level zapcore.LevelEnabler, stderr zapcore.WriteSyncer) *zap.Logger {
-	writer := zapcore.Lock(filtered(stderr))
-	levelEncoder := zapcore.CapitalLevelEncoder
-	if term.IsTerminal(int(os.Stdin.Fd())) {
-		levelEncoder = zapcore.CapitalColorLevelEncoder
+func init() {
+	zerolog.DisableSampling(true)
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnixMs
+	zerolog.CallerMarshalFunc = func(file string, line int) string {
+		short := file
+		for i := len(file) - 1; i > 0; i-- {
+			if file[i] == '/' {
+				short = file[i+1:]
+				break
+			}
+		}
+		file = short
+		return file + ":" + strconv.Itoa(line)
```

Comment:
I see this is from their examples, but we can probably use https://pkg.go.dev/path/filepath#Base for this?

```suggestion
		return filepath.Base(file) + ":" + strconv.Itoa(line)
```

Reply:
Yeah I took it verbatim without much thought. This is a much better implementation